File: perl/sha1.pl
#!/usr/bin/perl
use strict;

use Digest::SHA1  qw(sha1_hex);
my $msg = "clubhouse";
my $rounds = 5555555555;
for(my $i=1;$i<=$rounds;$i++){
  $msg = sha1_hex($msg);
}
print $msg,"\n";

File: nodejs/sha1.js
let crypto = require('crypto')
var msg = "clubhouse"
let rounds = 5_555_555_555

for (let i=1; i<=rounds; i++)
{
  var shasum = crypto.createHash('sha1')
  msg=shasum.update(msg).digest("hex")
}

console.log(msg)
